### ISSUE TITLE
Converting Countours to Labels

### ISSUE THREAD

**Issue by pakiessling** [FZJ-INM1-BDA/celldetection#12]
Hi, I must be overlooking something obvious.
How do I save calculated contours as a label image?

Keeping with your Coin example:
```
# Run model
with torch.no_grad():
    x = cd.to_tensor(img, transpose=True, device=device, dtype=torch.float32)
    x = x / 255  # ensure 0..1 range
    x = x[None]  # add batch dimension: Tensor[3, h, w] -> Tensor[1, 3, h, w]
    y = model(x)
```

Both` label = cd.data.contours2labels(y['contours'],size=img.shape[:2])` and` label = cd.data.contours2labels(np.array(y['contours']),size=img.shape[:2])` give me error messages.

**Comment by pakiessling** [comment]
Ok,

`label = cd.data.contours2labels(y["contours"][0].detach().numpy(),size=img.shape[:2])`

does the trick

**Comment by pakiessling** [comment]
I noticed that
`label = cd.data.contours2labels(y["contours"][0].to("cpu").detach().numpy(),size=img.shape[:2])`
returns multiple channels.
What excatly does "Channels are used to model overlap" mean?
Should I just use  `label = label[:,:,0]` for calculating scores, or something different?

**Comment by ericup** [comment]
Hi, for the `contours2labels` function it is important to note that it expects numpy arrays, rather than Tensors, and works on single batch items.
For specifics, you may also refer to the [documentation](https://docs.celldetection.org/en/latest/celldetection.data.html#celldetection.data.cpn.contours2labels).

By default, the label images come with channels, as contours may assign pixels to multiple objects.
Since such multi-assignments cannot be easily encoded in a channel-free label image, channels are used.
I updated the documentation to make this clearer.
There are many ways to resolve unwanted overlaps, i.e. channels. I just pushed `cd.data.resolve_label_channels`, which currently offers the solution used in the CellSeg challenge.

For your case you could do the following:
```python
contours, = cd.asnumpy(y['contours'])
labels = cd.data.contours2labels(contours, img.shape[:2])
labels = cd.data.resolve_label_channels(labels)  # remove channels
```

Note, that you need to install the latest version of `celldetection` from git to use `resolve_label_channels`.

**Comment by pakiessling** [comment]
Thanks for pushing such a quick update!
